pkgs/development/libraries/libusbsio/default.nix

```diff
@@ -1,4 +1,11 @@
-{ lib, stdenv, fetchzip, pkg-config, libusb1, systemdMinimal }:
+{ lib
+, stdenv
+, fetchzip
+, pkg-config
+, libusb1
+, systemdMinimal
+, darwin
+}:
 let
   binDirPrefix = if stdenv.isDarwin then "osx_" else "linux_";
 in
```

Comment:
Is this obsolete now?

Reply:
Yes, by passing an empty `BINDIR` in the makeFlags it forces the makefile to put the resulting binary in the `/bin/` folder rather than `/bin/{osx,linux}_$(uname -m)` folder, I figured this was cleaner than using the prefix.

Thanks for catching that one.